docs/working-with-projects/project-configuration/configure-webhooks.md

```diff
@@ -9,81 +9,120 @@ The action that is taken upon receiving a notification from a Harbor project dep
 
 ### Supported Events
 
-You can define one webhook endpoint per project. Webhook notifications provide information about events in JSON format and are delivered by `HTTP` or `HTTPS POST` to an existing webhhook endpoint URL that you provide. The following table describes the events that trigger notifications and the contents of each notification.
+You can define multiple webhook endpoints per project. Harbor supports two kinds of endpoints currently, One type is `HTTP`  and the other is `SLACK`. Webhook notifications provide information about events in JSON format and are delivered by `HTTP` or `HTTPS POST` to an existing webhhook endpoint URL or Slack address that you provide. The following table describes the events that trigger notifications and the contents of each notification.
```

Comment:
please change to  -> ’Harbor supports two kinds of endpoints currently, `HTTP` and `SLACK`. ‘